src/components/CVcomponents/HeaderCV.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import EditIcon from "../../images/edit_icon_orange.png"
+
+class HeaderCV extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      "isNameClicked": false,
+      "isJobClicked": false,
+      "isDescriptionClicked": false,
+      "name" : this.props.name,
+      "job" : this.props.job,
+      "description" : this.props.description
+    };
+
+    // This binding is necessary to make `this` work in the callback
+    this.nameClick = this.nameClick.bind(this);
+    this.jobClick = this.jobClick.bind(this);
+    this.descriptionClick = this.descriptionClick.bind(this);
+    this.saveNameInput = this.saveNameInput.bind(this);
+    this.saveJobInput = this.saveJobInput.bind(this);
+    this.saveDescriptionInput = this.saveDescriptionInput.bind(this);
+  }
+
+  //Functies die ervoor zorgen dat state veranderd veranderd naar false
+  nameClick() {
+    this.setState(state => ({
+      isNameClicked: !state.isNameClicked
+    }));
+  }
+
+  jobClick() {
+    this.setState(state => ({
+      isJobClicked: !state.isJobClicked}
+
+      ));
+  }
+
+  descriptionClick() {
+    this.setState(state => ({
+      isDescriptionClicked: !state.isDescriptionClicked
+    }));
+  }
+
+  //Functies die ervoor zorgt dat de input van het field wordt opgeslagen
+  saveNameInput(event) {
+    this.setState({name: event.target.value});
+    this.setState({isNameClicked: false})
+  }
+
+  saveJobInput(event) {
+    this.setState({job: event.target.value});
+    this.setState({isJobClicked: false})
+  }
+
+  saveDescriptionInput(event) {
+    this.setState({description: event.target.value});
+    this.setState({isDescriptionClicked: false})
+  }
+
+
+  render() {
+  return( <div className="headera4">
+                <div className="editable" onClick={this.nameClick}>
```

Comment:
This is indented 1 tab too many it seems. Or the first div should start on line 65 to make it fit.